packages/hast-util-from-webparser/index.ts

```diff
@@ -175,6 +185,10 @@ function getElementNameAndNS(name: string, implicitNs = false) {
   return { ns: info[0], name: info[1] }
 }
 
+function isGap(el: Node): boolean {
+  return el instanceof Text && !!el.value && !!el.value.match(/\n\s*?\n\s*?$/)
```

Comment:
Declare the regex as a `const` at the top in order to avoid reevaluation.

```js
return el instanceof Text && el.value && GAP_REGEX.test(el.value)
```